Add hint about event cleanup to Subscription doc (#13112)
kyma-project/kyma: docs/05-technical-reference/00-custom-resources/evnt-01-subscription.md

| **spec.filter.filters.eventSource.value** | Yes | Must be set to `""` for the NATS backend. |
| **spec.filter.filters.eventType.property** | Yes | Must be set to `type`. |
| **spec.filter.filters.eventType.type** | No | Must be set to `exact`. |

> | **spec.filter.filters.eventType.value** | Yes | Name of the event being subscribed to, for example: `sap.kyma.custom.commerce.order.created.v1`. The name should not contain any non-alphanumeric characters. Read [Event names](../evnt-01-event-names.md#event-name-cleanup) for more information. |

Reviewer: ```suggestion
| **spec.filter.filters.eventType.value** | Yes | Name of the event being subscribed to, for example: `sap.kyma.custom.commerce.order.created.v1`. The name cannot contain any non-alphanumeric characters. Read [Event names](../evnt-01-event-names.md#event-name-cleanup) for more information. |
```
